src/main/java/teammates/ui/output/FeedbackSessionLogType.java

```diff
@@ -0,0 +1,16 @@
+package teammates.ui.output;
+
+/**
+ * The feedback session log type.
+ */
+public enum FeedbackSessionLogType {
+    /**
+     * Visit Action for a Feedback Session.
+     */
+    VISIT,
+
+    /**
+     * Submission Action for a Feedback Session.
+     */
+    SUBMISSION,
+}
```

Comment:
Currently from `Const.FeedbackSessionLogTypes` we already have `ACCESS` and `SUBMISSION`, so there seems to be duplication with what we have over here. If we import those constants over and use them here through an enum constructor, such as `SUBMISSION(Const.FeedbackSessionLogTypes.SUBMISSION)`, would it be better?

Reply:
I have created a new enum `LogTypes` in the #11005  that is currently being used for generating types for the frontend. That can be reused here once that PR has been merged

Reply:
Have removed this for now, thanks!